src/components/VAccordion.vue

```diff
@@ -35,6 +35,7 @@
 
 <script>
 import SimpleArrowDown from '../components/icons/SimpleArrowDown'
+import 'focus-visible'
```

Comment:
I think we should import this at a higher level than this one specific icon, seems like it should be able to cover all focusable elements. At the very least it should be up in the accordion component where this icon actually becomes a focusable thing.

Reply:
then it would have to be imported on the client site not in the design system - right?

Reply:
i think we can just move it to the accordion for now. But this icon on its own isn't inherently a focusable control, the focus is happening to an instance of this icon and that instance's role as part of the accordion component.

I'm assuming Vue is smart enough here that this isn't getting loaded multiple times if it shows up in multiple places...

Reply:
ah nevermind i misread